doc/contributing.md

```diff
@@ -1,4 +1,5 @@
 % Include content from [../CONTRIBUTING.md](../CONTRIBUTING.md)
+
```

Comment:
This is a comment for the following include command, so I think it would be much clearer without a blank line.

Reply:
[MD031 - Fenced code blocks should be surrounded by blank lines](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md031---fenced-code-blocks-should-be-surrounded-by-blank-lines)

Reply:
This is a problem with MyST again - since the following is not a fenced code block, but a MyST command (and MyST commands are implemented through code blocks).